docs/general/dql/selects.rst

```diff
@@ -5,40 +5,46 @@
 Selecting data
 ==============
 
-Selecting (i.e. retrieving) data from CrateDB is done by using a SQL ``SELECT``
-statement. The response to a ``SELECT`` query contains the column names of the
-result, the actual result rows as a two-dimensional array of values, the row
-count and the duration.
+Selecting (i.e., retrieving) data from CrateDB can be done by using an SQL
+``SELECT`` statement. The response to a ``SELECT`` query includes the column
+names of the result, the result rows as a two-dimensional array of values, the
+row count, and the execution time.
+
+.. SEEALSO::
+
+    :ref:`SELECT syntax <sql_reference_select>`
 
 .. rubric:: Table of contents
 
 .. contents::
    :local:
 
+
 Introduction
 ============
 
 A simple select::
 
-    cr> select name, position from locations order by id limit 2;
-    +-------------------+----------+
-    | name              | position |
-    +-------------------+----------+
-    | North West Ripple | 1        |
-    | Arkintoofle Minor | 3        |
-    +-------------------+----------+
+    cr> select id, name from locations order by id limit 2;
+    +----+-------------------+
+    | id | name              |
+    +----+-------------------+
+    |  1 | North West Ripple |
+    |  2 | Outer Eastern Rim |
+    +----+-------------------+
     SELECT 2 rows in set (... sec)
 
 If the '*' operator is used, all columns defined in the schema are returned for
 each row::
 
-    cr> select * from locations order by id limit 1 offset 1;
-    +----+-------------------+--------------+--------+----------+---------...-+-------...-+-------------+
-    | id | name              |         date | kind   | position | description | race      | information |
-    +----+-------------------+--------------+--------+----------+---------...-+-------...-+-------------+
-    | 10 | Arkintoofle Minor | 308534400000 | Planet |        3 | Motivate... | {"desc... |        NULL |
-    +----+-------------------+--------------+--------+----------+---------...-+-------...-+-------------+
-    SELECT 1 row in set (... sec)
+    cr> select * from locations order by id limit 2;
+    +----+-------------------+--------------+--------+----------+--------------------------------------------------------------------------------------------------------------------------------------------------------------+-------------+---------------------------------------------------------------------------------------+-----------+
```

Comment:
Can we keep truncating the result table? I think without, readability gets worse.